modules/joda/README.md

```diff
@@ -18,28 +18,39 @@ libraryDependencies += "com.github.melrief" %% "pureconfig-joda" % "0.6.0"
 
 ## Example
 
+Define a Joda `DateTimeFormatter` for a style of writing dates which looks suspiciously like ISO 8601:
+
 ```scala
-// Define a Joda `DateTimeFormatter` for a style of writing dates which looks suspiciously like ISO 8601.
 import org.joda.time.format.DateTimeFormat
 val isoFormatter = DateTimeFormat.forPattern("yyyy-MM-dd'T'HH:mm:ss.SSSZ")
+```
 
-// Create a ConfigConvert to read DateTime with that format.
+Create a ConfigConvert to read DateTime with that format:
+```scala
+import pureconfig.loadConfig
 import pureconfig.module.joda.configurable._
+import com.typesafe.config.ConfigFactory.parseString
+
 implicit val dateTimeConverter = dateTimeConfigConvert(isoFormatter)
+```
 
-// An object to receive our configuration
+An object to receive our configuration;
+```scala
 import org.joda.time.DateTime
 case class GreatDatesConfig(apollo: DateTime, pluto: DateTime)
+```
 
-// We can read a GreatDatesConfig like:
-import pureconfig.loadConfig
-import com.typesafe.config.ConfigFactory.parseString
+We can read a GreatDatesConfig like:
+
+```scala
 val conf = parseString("""{
   apollo: "1969-07-20T20:18:00.000Z"
   pluto: "2021-01-20T06:59:59.999Z"
 }""")
+// conf: com.typesafe.config.Config = Config(SimpleConfigObject({"apollo":"1969-07-20T20:18:00.000Z","pluto":"2021-01-20T06:59:59.999Z"}))
+
 loadConfig[GreatDatesConfig](conf)
-// Success(GreatDatesConfig(1969-07-20T14:18:00.000-06:00,2021-01-19T23:59:59.999-07:00))
+// res1: Either[pureconfig.error.ConfigReaderFailures,GreatDatesConfig] = Right(GreatDatesConfig(1969-07-20T16:18:00.000-04:00,2021-01-20T01:59:59.999-05:00))
```

Comment:
Nitpick: It's sort of noisome that these dates are going to be rewritten each time the documentation is generated by someone with a different local time zone. Can you think of a way to avoid that? I've got nothing.